tests/ci/cdk/README.md

```diff
@@ -158,3 +159,57 @@ command.
  * `cdk deploy`      deploy this stack to your default AWS account/region
  * `cdk diff`        compare deployed stack with current state
  * `cdk docs`        open CDK documentation
+ 
+### Useful Docker image build commands
```

Comment:
All of the below commands are run as part of `run-cdk.sh` right? You may want to make a note that these commands replicate the steps that are performed in that script.

Reply:
Added.